processor/servicegraphprocessor/internal/store/store.go

```diff
@@ -135,3 +110,25 @@ func (s *store) Expire() {
 	for s.tryEvictHead() {
```

Comment:
This looks incomplete, but looking closely, it's fine. Perhaps add a comment, stating that it will iterate until there are no more items to be evicted?

Reply:
Good point, added in c182bebcf7306f152b153623b0fdc97b3648e2b3